tests/core/block/e2e/test_correctness.py

```diff
@@ -54,6 +43,11 @@ def test_v1_v2_greedy_equality_with_preemption(baseline_llm_generator,
 
     NOTE: We want a significant number of generated tokens so that any incorrect
     KV mapping has time to build up error.
+
+    NOTE(Kuntai): Though we have removed block manager v1, this test is still
+    useful as it asserts the behavior of block manager v2 (now it is called 
+    SelfAttnBlockManager) is the same when swapping / preemption, so we keep 
+    this test.
```

Comment:
Then maybe just rename the test to be something like `test_block_manager_with_preemption`?

Reply:
Fixed